### DISCUSSION TITLE
JAlbum

### DISCUSSION THREAD

**Discussion by smolesen** [photoprism/photoprism#3274] [Q&A]
Hi

Jut wondering if there exists some kind of integration to JAlbum? I think Photosrism is awesome for managing images, but for web-albums JAlbum is better.
Would be awesome if you could somehow "export" your albums from Photoprism into JAlbum

/Søren

**Comment by iammartian** [comment]
Ok I'll try doing it.
Should I do this with JAlbum ?

With Regards,
Maverick

**Comment by lastzero** [comment]
JAlbum does not seem to be open source? In this case a direct integration is impossible. However, we would like to implement export of statically generated albums (based on HTML, CSS and JS). These could use any available JS library. Similar to other static site generators like Hugo (which should also have a gallery): https://gohugo.io/
